apm/event.go

```diff
@@ -37,18 +37,22 @@ func (e *eventWindow) Message() message.Composer {
 	}
 
 	type output struct {
-		Operation string  `bson:"operation" json:"operation" yaml:"operation"`
-		Duration  float64 `bson:"duration_secs" json:"duration_secs" yaml:"duration_secs"`
-		Succeeded int64   `bson:"succeeded" json:"succeeded" yaml:"succeeded"`
-		Failed    int64   `bson:"failed" json:"failed" yaml:"failed"`
+		Duration   float64 `bson:"duration_secs" json:"duration_secs" yaml:"duration_secs"`
+		Succeeded  int64   `bson:"succeeded" json:"succeeded" yaml:"succeeded"`
+		Failed     int64   `bson:"failed" json:"failed" yaml:"failed"`
+		Database   string  `bson:"database" json:"database" yaml:"database"`
+		Collection string  `bson:"collection" json:"collection" yaml:"collection"`
+		Command    string  `bson:"command" json:"command" yaml:"command"`
 	}
 	colls := make([]output, 0, len(e.data))
 	for k, v := range e.data {
 		colls = append(colls, output{
```

Comment:
can we also leave the operation in this struct in addition to breaking things out

Reply:
done